### PR TITLE
feat: deploy vCluster on AKS

### PR DESCRIPTION
**Pull request by Piotr1215**
Closes DOC-350

[Deploy vCluster on AKS Preview](https://deploy-preview-367--vcluster-docs-site.netlify.app/docs/vcluster/next/deploy/environment/aks)

### CODE REVIEW

File: vcluster/deploy/environment/aks.mdx [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    --node-vm-size $NODE_SIZE \
+    --enable-managed-identity \
+    --enable-addons monitoring \
+    --generate-ssh-keys
+```
+
+This command creates an AKS cluster with managed identity enabled and Azure Monitor for containers.
+
+### Configure kubectl
+
```
Comment (deniseschannon) [inline vcluster/deploy/environment/aks.mdx:67]: I just merged the EKS docs, but how come we don't give docs on how to configure kubectl for EKS?

File: vcluster/deploy/environment/aks.mdx [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+```
+NAME                                STATUS   ROLES    AGE   VERSION
+aks-nodepool1-34960941-vmss000000   Ready    <none>   67m   v1.29.9
+aks-nodepool1-34960941-vmss000001   Ready    <none>   67m   v1.29.9
+```
+
+
+## Create virtual cluster
+
+Create a virtual cluster using the CLI:
```
Comment (deniseschannon) [inline vcluster/deploy/environment/aks.mdx:92]: Why wouldn't this have matched the wording/text as create virtual cluster in the EKS docs?

File: vcluster/deploy/environment/aks.mdx [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+```bash title="Create virtual cluster"
```
Comment (deniseschannon) [inline vcluster/deploy/environment/aks.mdx:94]: Can we get the EKS docs to match the title of this command?

File: vcluster/deploy/environment/aks.mdx [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+
+You should see output similar to:
+```
+NAME                                                   READY   STATUS    RESTARTS   AGE
+coredns-666d64755b-k5njg-x-kube-system-x-my-vcluster   1/1     Running   0          3m11s
+my-vcluster-0                                          1/1     Running   0          6m33s
+```
+
+This configuration ensures that:
+- Service accounts are properly synced between virtual and host clusters
```
Comment (deniseschannon) [inline vcluster/deploy/environment/aks.mdx:114]: I didn't catch this in the EKS docs, but what did we do to ensure this?

**Review by deniseschannon — CHANGES_REQUESTED** [review]

File: vcluster/deploy/environment/aks.mdx (hunk 1 above)
Comment (Piotr1215) [reply vcluster/deploy/environment/aks.mdx:67]: There is no special configuration needed and `kubectl` is [mentioned in the EKS prerequisites](https://www.vcluster.com/docs/vcluster/next/deploy/environment/eks#prerequisites).

File: vcluster/deploy/environment/aks.mdx (hunk 2 above)
Comment (Piotr1215) [reply vcluster/deploy/environment/aks.mdx:92]: They should be the same now, I'll make a `_partial`  out of it.

File: vcluster/deploy/environment/aks.mdx (hunk 4 above)
Comment (Piotr1215) [reply vcluster/deploy/environment/aks.mdx:114]: This should actually be moved below `Create a virtual cluster with managed identity`.

**Review by deniseschannon — APPROVED** [review]

**Pull request merged**